kedro-datasets/tests/spark/test_spark_streaming_dataset.py

```diff
@@ -4,9 +4,11 @@
 import pytest
 from kedro.io.core import DataSetError
 from moto import mock_s3
+from pyspark import __version__
 from pyspark.sql import SparkSession
 from pyspark.sql.types import IntegerType, StringType, StructField, StructType
 from pyspark.sql.utils import AnalysisException
+from semver import VersionInfo
```

Comment:
Wondering if we should use `packaging` for this, instead of `semver`? https://packaging.pypa.io/en/stable/version.html (see rationale in https://github.com/kedro-org/kedro-viz/issues/1460